src/plugins.js

```diff
@@ -0,0 +1,131 @@
+export default {
+  // es2015
+  "transform-es2015-arrow-functions": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-block-scoped-functions": {
+    edge: 11,
+    firefox: 46,
+    chrome: 41,
+    opera: 28,
+  },
+  "transform-es2015-block-scoping": {
```

Comment:
Some code that builds with the transform will fail at runtime with native block scoping, but in this case it's the user's fault for not respecting the TDZ.